doc/source/user_guide/io.rst

```diff
@@ -5477,6 +5478,40 @@ web site.
 
 No official documentation is available for the SAS7BDAT format.
 
+.. _io.spss:
+
+.. _io.spss_reader:
+
+SPSS formats
+-----------
+
+The top-level function :func:`read_spss` can read (but not write) SPSS
+`sav` (.sav) and  `zsav` (.zsav) format files(since *v0.25.0*).
+
+SPSS files contain column names. By default the
+whole file is read and returned as a ``DataFrame`` with all columns.
+
+Specify a ``usecols`` to obtain a subset of columns. Specify ``apply_value_formats=True``
```

Comment:
Isn't this called `convert_categoricals`? Also I think worth mentioning this is the default so user would have to disable if they didn't want it

Reply:
Thanks for the good catch! Totally agree.